teachertool/src/state/actions.ts

```diff
@@ -62,24 +62,12 @@ type HideModal = ActionBase & {
  * Union of all actions
  */
 
-export type Action =
-    | PostNotification
-    | RemoveNotification
-    | SetProjectMetadata
-    | SetEvalResult
-    | SetTargetConfig
-    | SetCatalog
-    | SetSelectedCriteria
-    | RemoveCriteriaInstance
-    | ShowModal
-    | HideModal;
+export type Action = PostNotification | RemoveNotification | SetProjectMetadata | SetEvalResult | SetTargetConfig | SetCatalog | SetSelectedCriteria | RemoveCriteriaInstance | ShowModal | HideModal;
```

Comment:
Could we `// prettier-ignore` both of the export statements in this file? I realize these might fit within our expanded max line length, but I feel like it's okay to make an exception here...

Reply:
Certainly. There is a tension with increased line width vs what we'd prefer to have stacked. We may want to bring the max line width in a bit. 200 is very wide.

Reply:
Setting width to 120 corrected this. And the results seemed more reasonable to me in general.

Reply:
Seems like you found the sweet spot!